client/src/components/grants/List.tsx

```diff
@@ -115,26 +119,37 @@ function ProjectsList() {
     }
   }, [projectsQueryResult?.projects.length, roundToApply]);
 
-  const projectId = projectsQueryResult?.projects[0]?.id;
-
-  const hasUserAppliedToRouond = async (): Promise<boolean> => {
-    const rounds = await fetchIfUserHasAppliedToRound(
-      roundManagerClient!,
-      projectId!
-    );
+  useEffect(() => {
+    const hasUserAppliedToRouond = async (): Promise<boolean> => {
+      const projectId = projectsQueryResult?.projects[0]?.id;
+      if (projectId) {
+        console.log("projectId", projectId);
+        await fetchIfUserHasAppliedToRound(
```

Comment:
can someone look at this and see why it keeps returning null when I know it should have value @DanieleSalatti